packages/dotcom-ui-header/src/components/drawer/topLevelPartials.tsx

```diff
@@ -136,7 +139,11 @@ const SectionTertiary = (props: TNavMenuItem) => (
       const divideItem = index === 0 ? 'o-header__drawer-menu-item--divide' : ''
 
       return (
-        <li key={item.url} className={`o-header__drawer-menu-item ${divideItem}`}>
+        <li
+          data-component={divideItem ? 'drawer-menu-item--divide' : undefined}
```

Comment:
Suggestion: as per the previous question
```suggestion
          data-component='drawer-menu-item'
```

Reply:
Here we are targeting only `o-header__drawer-menu-item--divide` items. For the Workspace menu we need only the last `o-header__drawer-menu-item--divide` item

Reply:
I would rather not rely on the divider but on the item themselves. What do you think about something similar to this:

```
document.querySelectorAll('.o-header__drawer-menu-item  [data-trackable="Portfolio"]')
```
With this targeting, we can clone from the Portfolio item and append before it and you will not need the modifier `--divide`

Reply:
We can not rely on individual items because it's up to navigation API to return data. We can not be sure Portfolio will exist in the future as an item. The `--divide` is used to insert the Workspace link before the last divider so I don't see how we could target this in any other way

Reply:
> We can not rely on individual items because it's up to navigation API to return data
Correct. However, this solution is meant to be temporary. At some point we want this menu to be part of the navigation and be rendered server-side only for B2B users. 
I'm happy to keep this as is 👍